Source/WebCore/css/parser/CSSPropertyParser.cpp

```diff
@@ -2546,6 +2546,44 @@ bool CSSPropertyParser::consumeListStyleShorthand(bool important)
     return m_range.atEnd();
 }
 
+bool CSSPropertyParser::consumeWhiteSpaceShorthand(bool important)
+{
+    auto ident = consumeIdentRaw(m_range);
+    if (!ident)
+        return false;
+    if (!m_range.atEnd())
+        return false;
```

Comment:
This can be removed, this check is done elsewhere.

Reply:
It seems like leaving out this check causes `imported/w3c/web-platform-tests/css/css-text/parsing/white-space-invalid.html` to fail.